### DISCUSSION TITLE
How to intall a linux tar file on chromebook 311 for the purpose of using discord desktop app

### DISCUSSION THREAD

**Discussion by WatchingOracle** [obsproject/obs-studio#4493] [Development Questions] [answered]
Hello,
This is my first discussion. I'll start with a Namaste. So, if you are trying to play games on discord with a chromebook 311 you will keep running into the Linux tar file for desktop as an option.  Chromebook uses the chrome browser as its operating system.  It is an application based operating system.  It must be possible to install the desktop app but the chromebook 311 requires an unarchiver application to accomplish this.

**Comment by RytoEX** [comment, marked as answer]
Sorry, but we don't support Chromebooks.

**Comment by Gol-D-Ace** [comment]
Neither are we Discord support.
`purpose of using discord desktop app`
